colin-api/src/colin_api/models/reset.py

```diff
@@ -239,3 +239,49 @@ def reset_filings(cls, start_date: str = None, end_date: str = None, identifiers
             if con:
                 con.rollback()
             raise err
+
+    @classmethod
+    def reset_filings_by_event(cls, event_ids: list = []):
```

Comment:
Besides the one entry for corp_party(`Party.reset_dirs_by_events(cursor=cursor, event_ids=event_ids`) that I was wondering about, I think all the other commented out reset calls are not relevant to ARs from what I can see.

Can we remove these lines from the code?

Reply:
Sure